src/serlo-editor/plugins/rows/components/add-row-button-floating.tsx

```diff
@@ -25,7 +25,7 @@ export function AddRowButtonFloating({
           <button
             className={clsx(
               tw`
-                add-trigger z-[70] h-7 rounded-full py-2 text-gray-600 transition-all
+                add-trigger z-[20] h-7 rounded-full py-2 text-gray-600 transition-all
```

Comment:
tiny thing: for 20 you can use the default
```suggestion
                add-trigger z-20 h-7 rounded-full py-2 text-gray-600 transition-all
```